core/config/locales/en.yml

```diff
@@ -577,6 +577,10 @@ en:
           attributes:
             user_id:
               default_address_exists: "already has a default address"
+        spree/wallet_payment_source:
+          attributes:
+            payment_source:
+              has_to_be_payment_source_class: "has to be a Spree::PaymentSource"
```

Comment:
Not sure yet, mostly here to solve the missing translation error we got in the model when the validation fails.